packages/contracts/scripts/program/deployProgramImplementation.ts

```diff
@@ -15,10 +15,10 @@ export async function main() {
   });
 
   // Deploy ProgramImplementation 
-  const contractFactory = await ethers.getContractFactory("ProgramImplementation");
+  const contractFactory = await ethers.getContractFactory("DummyProgramImplementation");
```

Comment:
why was this renamed to `DummyProgramImplementation` ?

Reply:
```suggestion
  const contractFactory = await ethers.getContractFactory("ProgramImplementation");
```